test/functional/base_test.go

```diff
@@ -296,3 +296,37 @@ func CinderVolumeNotExists(name types.NamespacedName) {
 		g.Expect(k8s_errors.IsNotFound(err)).To(BeTrue())
 	}, timeout, interval).Should(Succeed())
 }
+
+// GetExtraMounts - Utility function that simulates extraMounts pointing
+// to a Ceph secret
+func GetExtraMounts() []map[string]interface{} {
```

Comment:
wondering if it would make sense and help others to add a test helper to the storage module in https://github.com/openstack-k8s-operators/lib-common/blob/main/modules/storage or is it to specific for the component?

Reply:
This function is specific to the component,it  only craft a piece of the CR that is applied beforehand, and later checked within a dedicated envTest.
I never thought about providing a `lib-common/modules/storage` helper, but this would definitely be an interesting idea that would help to:
1. simplify the operators test logic when need to validate an `extraMount`
2. provide a wrapper to perform the assertion and hide the detailed steps required to get the Volume/VolumeMount and call the existing assert

Perhaps I can do that in a follow up so I can try to add such helper in `lib-common` first.

Reply:
sure, doing this later in a follow up whenever time permits is fine